### PR TITLE
Feature/migrate new landing

### PR DESCRIPTION
**Pull request by DemogorGod**
@natemiller1 @shanejearley 

Just a pr for pushing the new landing page up it's excatly as Hawar had built it, and the blog post is done it just needs styling review/adjustments 

this is what it looks like right now:
Screenshot 2023-10-04 at 9 53 54 AM

in order to get the blog to run right now need to run this command in a separate terminal than loading the landing page 
(Shane will handle those fixes)
`npm run dev:blogs`

### CODE REVIEW

**Comment by natemiller1** [comment]
@DemogorGod If I make changes to the blog on the hackmd side of things how will that get pushed to the front end?

**Comment by DemogorGod** [comment]
> @DemogorGod If I make changes to the blog on the hackmd side of things how will that get pushed to the front end?
> 
> Blog looks great though!

@natemiller1 
Yes, all you need to do is update hackmd notes, make sure to publish them and include the blog tag on  them as well

File: apps/landing/src/pages/changelog/Changelog.vue [hunk 1]
```diff
@@ -0,0 +1,164 @@
+<script lang="ts" setup>
+import snarkdown from 'snarkdown'
+
```
Comment (ccali11) [inline apps/landing/src/pages/changelog/Changelog.vue:3]: so much space between lines...

File: apps/web/src/pages/overview/Overview.vue
Comment (ccali11) [inline apps/web/src/pages/overview/Overview.vue:1]: Will cause conflicts with all of the refactoring I did with the `useAuth` composable, but I'll help resolve them when it comes time to merge my `multi-account-management` PR.

File: package.json [hunk 2]
```diff
@@ -54,11 +55,14 @@
     "eslint-plugin-vue": "^8.7.1",
     "esno": "^0.16.3",
     "ethers": "^5.7.2",
-    "express": "^4.18.1",
+    "express": "^4.18.2",
```
Comment (ccali11) [inline package.json:58]: I don't think these dependencies belong here. Put them in their respective directories/packages using `npm i <external-package> --workspace @casimir/<our-package>`

File: package.json [hunk 3]
```diff
@@ -80,5 +84,8 @@
     "hooks": {
       "after:bump": "npx auto-changelog -p"
     }
+  },
+  "dependencies": {
+    "snarkdown": "^2.0.0"
```
Comment (ccali11) [inline package.json:89]: (this belongs in `@casimir/landing`)

File: scripts/root/dev.ts [hunk 4]
```diff
@@ -1,166 +1,218 @@
 import { ethers } from 'ethers'
 import { loadCredentials, getSecret } from '@casimir/aws'
-import { ETHEREUM_CONTRACTS, ETHEREUM_NETWORK_NAME, ETHEREUM_RPC_URL } from '@casimir/env'
+import {
```
Comment (ccali11) [inline scripts/root/dev.ts:3]: Why the update to this line?

File: scripts/root/dev.ts
Comment (ccali11) [inline scripts/root/dev.ts:1]: A little hard to follow this file; I think we did this together and I know what changes you made, but it shows a lot of changes where I don't think there were that many. If that's the case, then the file is probably just fine. 👍 Correct me if I'm wrong.

File: services/blogs/package.json [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "npx esno -r dotenv/config src/index.ts"
+  },
+  "author": "Stevan Najeeb",
+  "license": "ISC",
+  "dependencies": {
+    "cors": "^2.8.5",
+    "express": "^4.18.2"
```
Comment (ccali11) [inline services/blogs/package.json:14]: This `express` dependency is where it belongs, so just uninstall the one (and any other rogue packages) sitting in the `package.json` file at the project level.

File: services/blogs/src/index.ts [hunk 6]
```diff
@@ -0,0 +1,74 @@
+import express from 'express'
+import cors from 'cors'
+
+const app = express()
+const PORT = 3003
+app.use(cors())
+app.use(express.json())
+
+app.get('/api/hackmd', async (req, res) => {
+  const token = '4DXXJDRZGGRFTYTV1LH633U66W32PGKTKF6NPEQKPGNEO5TJXJ'
```
Comment (ccali11) [inline services/blogs/src/index.ts:10]: If this is a private token, we'll need to create a new HackMD account and throw this in an environment variable.

File: services/blogs/src/index.ts [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+
+  const requestOptions = {
+    method: 'GET',
+    headers: {
+      Authorization: `Bearer ${token}`,
+      'Content-Type': 'application/json',
+    },
+  }
+  // lO0eXsOqSOqOp6p8r61GIA
```
Comment (ccali11) [inline services/blogs/src/index.ts:19]: What is this guy? lol

File: services/blogs/src/index.ts [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+
+  if (!response.ok) {
+    console.error(
+      `Express HackMD API Error Fetching Team Notes: ${response.status} ${response.statusText}`
+    )
+    res.status(response.status).send('API Error')
+    return
+  }
+
+  const data = await response.json()
```
Comment (ccali11) [inline services/blogs/src/index.ts:33]: Can combine these two lines: `const filteredData = (await response.json()).filter(()=>{})`

File: services/blogs/src/index.ts [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+  if (!response.ok) {
+    console.error(
+      `Express HackMD API Error Fetching Team Note: ${response.status} ${response.statusText}`
+    )
+    res.status(response.status).send('API Error')
+    return
+  }
+
+  const data = await response.json()
+  console.log('data:', data)
```
Comment (ccali11) [inline services/blogs/src/index.ts:68]: Can remove.

**Review by ccali11 — CHANGES_REQUESTED** [review]
@DemogorGod - I didn't think I saw you request me to review yesterday, but saw it this morning and ran through it. There's a couple things to change; just follow the comments and reach out on Slack if you need any clarification.

**Comment by DemogorGod** [comment]
Screenshot 2023-10-05 at 11 54 54 AM
Screenshot 2023-10-05 at 11 55 01 AM

File: package.json [hunk 10]
```diff
@@ -54,11 +55,13 @@
     "eslint-plugin-vue": "^8.7.1",
     "esno": "^0.16.3",
     "ethers": "^5.7.2",
-    "express": "^4.18.1",
     "http-proxy-middleware": "^2.0.6",
     "husky": "^8.0.1",
     "lint-staged": "^13.0.0",
     "release-it": "^15.6.0",
+    "sass": "^1.68.0",
```
Comment (ccali11) [inline package.json:62]: @DemogorGod - move these to landing page `package.json`

File: services/blogs/src/index.ts (hunk 6 above)
Comment (ccali11) [reply services/blogs/src/index.ts:10]: need `.env` variable for this?

**Review by ccali11 — CHANGES_REQUESTED** [review]
@DemogorGod - Take a quick look at my two most recent comments before merging:

1. package.json dependencies 
2. the token we talked about

File: apps/landing/package.json [hunk 11]
```diff
@@ -11,6 +11,8 @@
   "dependencies": {
     "@heroicons/vue": "^1.0.6",
     "iconoir": "^5.3.2",
+    "sass": "^1.69.1",
```
Comment (ccali11) [inline apps/landing/package.json:14]: nice! 👍

File: apps/web/package.json [hunk 12]
```diff
@@ -18,6 +18,8 @@
     "d3": "^7.8.1",
     "ethers": "^5.7.2",
     "feather-icons": "^4.29.0",
+    "sass": "^1.69.1",
+    "snarkdown": "^2.0.0",
```
Comment (ccali11) [inline apps/web/package.json]: We don't use `snarkdown` in the `web` app.

File: package.json [hunk 13]
```diff
@@ -54,11 +55,12 @@
     "eslint-plugin-vue": "^8.7.1",
     "esno": "^0.16.3",
     "ethers": "^5.7.2",
-    "express": "^4.18.1",
     "http-proxy-middleware": "^2.0.6",
     "husky": "^8.0.1",
     "lint-staged": "^13.0.0",
     "release-it": "^15.6.0",
+    "vite-plugin-md": "^0.22.5",
```
Comment (ccali11) [inline package.json:62]: These two gotta move too, right?

File: apps/web/package.json (hunk 12 above)
Comment (ccali11) [reply apps/web/package.json]: I will address this and approve.

**Review by ccali11 — APPROVED** [review]
Approving but still have to address hackmd token + make sure root `package.json` is clean.
